src/supported-value.js

```diff
@@ -11,13 +12,27 @@ const transitionProperties = [
 ]
 const transPropsRegExp = /(^\s*\w+)|, (\s*\w+)/g
 let el
+let isDoubleArray = false
+let isArray = false
+let isImportantExist = false
 
 function prefixTransitionCallback(match, p1, p2) {
   if (p1 === 'all') return 'all'
   if (p2 === 'all') return ', all'
   return p1 ? supportedProperty(p1) : `, ${supportedProperty(p2)}`
 }
 
+function stringifyValue(value) {
```

Comment:
why do you need this? in order to get a css value string without important you can use toCssValue(value, true), see the implementation there